### PR TITLE
Cors bug fix

### PR DESCRIPTION
**Pull request by patricksimonian**
The cors config was not being parsed for json !

### CODE REVIEW

**Review by ShovelHand — COMMENTED** [review]
Readability is improved. The only thing I'm not sure about is the name getJSONFromBufferWithError; I was trying to figure out where the error was happening that leads us to this method, but it looks like it's  more to do with the catch block with error logging. Also the comment "Converts a Buffer to JSON but prints out a friendly log error message saying what file error'd" makes it sound like an error is expected. Is that the case?

File: api/src/utils/config.js [hunk 1]
```diff
@@ -2,7 +2,24 @@ import { readFileSync } from 'fs'
...
+ * Converts a Buffer to JSON but prints out a friendly log error message saying what file error'd
+ * @param {String} fileName
+ * @param {Buffer} data
+ * @returns {Object}
+ */
+export const getJSONFromBufferWithError = (fileName, data) => {
+  try {
+    return JSON.parse(data.toString())
+  } catch (e) {
+    console.log('THIS WAS CALLED')
```
Comment (patricksimonian) [inline api/src/utils/config.js:15]:
```suggestion
```

Comment (patricksimonian) [inline api/src/utils/config.js:15]:
```suggestion
```

**Comment by patricksimonian** [comment]
> Readability is improved. The only thing I'm not sure about is the name getJSONFromBufferWithError; I was trying to figure out where the error was happening that leads us to this method, but it looks like it's more to do with the catch block with error logging. Also the comment "Converts a Buffer to JSON but prints out a friendly log error message saying what file error'd" makes it sound like an error is expected. Is that the case?

good feedback. I've made changes

**Pull request merged**